src/components/category/catGroup.js

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
 import CatItem from './catItem';
 
 
 const catGroup = (props) => {
   let items = props.items.map(item => {
+    if(!item.name) return false;
```

Comment:
```suggestion
    if(item.name)
```

would be this better?

Reply:
just return the ones with  name instead of returning false, maybe im wrong, test it first.